.gitignore

```diff
@@ -57,7 +57,11 @@ docs/_build/
 target/
 
 .env
+.venv
+
+# IDE things
 .idea
+.vscode
```

Comment:
There are some projects I work on where sharing the configuration is useful to ensure people have uniform tooling and the like (eg prompting them to install extensions when they open the project), so I'd rather not disable it globally. I still think it's a good idea to keep it in this repo too, since it doesn't do any harm.